packages/datadog-instrumentations/src/cucumber.js

```diff
@@ -0,0 +1,82 @@
+'use strict'
+
+const { addHook, channel } = require('./helpers/instrument')
+const shimmer = require('../../datadog-shimmer')
+
+const runStartCh = channel('ci:cucumber:run:start')
+const runEndCh = channel('ci:cucumber:run:end')
+const runAsyncEndCh = channel('ci:cucumber:runAsync:end')
```

Comment:
Channel names should be the names of operations and not functions.

Reply:
something like `ci:cucumber:run:async-end`?